internal-services/catalog/pulp-push-disk-images-task.yaml

```diff
@@ -197,25 +197,28 @@ spec:
         process_component_for_developer_portal() { # Expected argument are [component json], [content_directory]
           COMPONENT=$1
 
-          productName=$(jq -er '.contentGateway.productName' <<< "${COMPONENT}") \
-            || (echo Missing contentGateway.productName value for component. This should be an existing product in the \
-                Developer Portal. Failing && exit 1)
-          productCode=$(jq -er '.contentGateway.productCode' <<< "${COMPONENT}") \
-            || (echo Missing contentGateway.productCode value for component. This should be an existing product in the \
-                Developer Portal. Failing && exit 1)
-          productVersionName=$(jq -er '.contentGateway.productVersionName' <<< "${COMPONENT}") \
-            || (echo Missing contentGateway.productVersionName value for component. This should be an existing product \
-                in the Developer Portal. Failing && exit 1)
-          filePrefix=$(jq -er '.contentGateway.filePrefix' <<< "${COMPONENT}") \
-            || (echo Missing contentGateway.filePrefix value for component. This should be the prefix for files to \
-                upload to the Developer Portal. Failing && exit 1)
+          productName="$(jq -er '.contentGateway.productName // ""' <<< "${COMPONENT}")" \
```

Comment:
I don't think the ` // ""` does anything here as you are failing if the field doesn't exist.. not using a default, but it doesn't break anything having it there. Up to you if you want to remove it (same for all 3)

Reply:
nice catch. fixed